### PR TITLE
Add dynamic dates

### PR DESCRIPTION
**Pull request by 23nith**
chrome_Gx34cy6IqV

### CODE REVIEW

File: api/addToReadingList.ts [hunk 1]
```diff
@@ -13,7 +15,7 @@ export const addToReadingList = (
       user_id: userID,
       rating: 0,
       notes: "",
-      start_date: "",
+      start_date: utc,
```
Comment (normancapule) [inline api/addToReadingList.ts]: What's `start_date` for?

File: pages/book.tsx [hunk 2]
```diff
@@ -22,6 +22,42 @@ const book = () => {
   const currentBook: IBook = previousPage != "/discover" ? list.book : book;
   const router = useRouter();
 
+  const startDate = new Date(list?.start_date);
```
Comment (normancapule) [inline pages/book.tsx]: You can make your life easier by adding node packages like date-fns or moment

File: api/addToReadingList.ts (hunk 1 above)
Comment (23nith) [reply api/addToReadingList.ts]: This is to display the start date on the books detail page.

Comment (normancapule) [reply api/addToReadingList.ts]: Yeah - but what does the start date do?

Comment (23nith) [reply api/addToReadingList.ts]: It determines when the user placed the book on the reading list.

File: pages/book.tsx [hunk 3]
```diff
@@ -22,6 +23,18 @@ const book = () => {
   const currentBook: IBook = previousPage != "/discover" ? list.book : book;
   const router = useRouter();
 
+  const startMonth: string = moment().month(list?.created_at).format("MMM");
+  const startDay: string = moment().date(list?.created_at).format("DD");
```
Comment (normancapule) [inline pages/book.tsx]: iirc this will work

moment(list.created_at).format("MMM DD")

**Review by normancapule — APPROVED** [review]
Much simpler nice

File: pages/book.tsx [hunk 4]
```diff
@@ -23,16 +23,12 @@ const book = () => {
...
-  const start_date_formatted: string = startMonth + " " + startDay;
+  const start_date_formatted: string = moment(list.created_at).format("MMM DD");
 
   let finish_date_formatted: string;
 
   if (list?.finish_date) {
-    const finishMonth: string = moment().month(list?.finish_date).format("MMM");
-    const finishDay: string = moment().date(list?.finish_date).format("DD");
-    finish_date_formatted = finishMonth + " " + finishDay;
+    finish_date_formatted = moment(list.created_at).format("MMM DD");
```
Comment (normancapule) [inline pages/book.tsx:31]: pst shouldn't this be `list?.finish_date`? :D

Comment (23nith) [reply pages/book.tsx:31]: Oops, okay got it already corrected on a new PR. Thank you!

**Pull request merged**